lib/authn.js

```diff
@@ -24,6 +24,11 @@ export async function openDatabase(options) {
     ...options
   };
 
+  // if the user specified a connection URL use it
```

Comment:
This is confusing ... because it's a NOT conditional. Does this mean?:

```suggestion
  // if a `url` was not specified, create one from the `config`
```